src/gtk/dataview.cpp

```diff
@@ -3814,9 +3814,16 @@ bool wxDataViewCtrlInternal::EnableDragSource( const wxDataFormat &format )
     return true;
 }
 
-bool wxDataViewCtrlInternal::EnableDropTarget( const wxDataFormat &format )
+bool wxDataViewCtrlInternal::EnableDropTarget( const wxDataFormatArray& formats )
 {
-    wxGtkString atom_str( gdk_atom_name( format  ) );
+    if (formats.GetCount() == 0)
+    {
+        gtk_tree_view_unset_rows_drag_dest(GTK_TREE_VIEW(m_owner->GtkGetTreeView()));
+
+        return true;
+    }
+
+    wxGtkString atom_str( gdk_atom_name( formats.Item(0) ) );
     m_dropTargetTargetEntryTarget = wxCharBuffer( atom_str );
 
     m_dropTargetTargetEntry.target =  m_dropTargetTargetEntryTarget.data();
```

Comment:
Here we can create C-style `GtkTargetEntry` array instead of single `m_dropTargetTargetEntry` object and pass it to `gtk_tree_view_enable_model_drag_dest` call.